client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/SubmissionEditForm.jsx

```diff
@@ -33,11 +33,14 @@ import {
 import SubmissionAnswer from '../../components/SubmissionAnswer';
 import QuestionGrade from '../../containers/QuestionGrade';
 import GradingPanel from '../../containers/GradingPanel';
-import Comments from '../../containers/Comments';
 import { formNames, questionTypes } from '../../constants';
 import translations from '../../translations';
 import submissionFormValidate from './submissionFormValidate';
 
+const Comments = React.lazy(() =>
+  import(/* webpackChunkName: "comment" */ '../../containers/Comments'),
```

Comment:
Do we need to modify the webpack config for this?

Reply:
No need for that since we already have the magic webpackchunk comment